docat/app.py

```diff
@@ -9,17 +9,19 @@
 """
 
 import os
+import secrets
 from http import HTTPStatus
-from pathlib import Path
 
 from flask import Flask, request, send_from_directory
+from tinydb import Query, TinyDB
 from werkzeug.utils import secure_filename
 
-from docat.docat.utils import create_nginx_config, create_symlink, extract_archive
+from docat.docat.utils import UPLOAD_FOLDER, create_nginx_config, create_symlink, extract_archive, remove_docs
 
 app = Flask(__name__)
-app.config["UPLOAD_FOLDER"] = os.getenv("DOCAT_DOC_PATH", "/var/docat/doc")
+app.config["UPLOAD_FOLDER"] = os.getenv("DOCAT_DOC_PATH", UPLOAD_FOLDER)
 app.config["MAX_CONTENT_LENGTH"] = 100 * 1024 * 1024  # 100M
+app.db = TinyDB('db.json')
 
 
 @app.route("/api/<project>/<version>", methods=["POST"])
```

Comment:
Schould we autoclaim on the first uppload? Or is this not obvious enough?

Reply:
Also should we also add (maybe optional) token authentication for upploads?

Reply:
we could add a protected mode where you need the token to change anything - but i would not add it by default